packages/freezed/lib/src/templates/concrete_template.dart

```diff
@@ -370,41 +367,123 @@ class Property {
       );
     }
 
+    final jsonKeyValues = element.jsonKeyValues;
+    if (defaultValue != null &&
+        jsonKeyValues != null &&
+        jsonKeyValues.containsKey('defaultValue')) {
+      throw InvalidGenerationSourceError(
+        '@Default cannot be used in conjunction with @JsonKey(defaultValue:)',
```

Comment:
Why is this needed?

Reply:
So I added this for safety, to prevent someone writing something like this:
```dart
factory User({@Default(42) @JsonKey(defaultValue: 30) int age})
```
At the call site, it is unclear what behavior will result from this.